app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimDatePage.scala.html

```diff
@@ -80,9 +80,9 @@
     }
 }
 
+@defining(LocalDate.now().getYear - 1) { dynamicYear =>
+@defining(s"$dynamicYear") { dynamicHintDate =>
```

Comment:
is this just to cast to string? dynamicYear would work. There is also .toString on everything. Though a lot of Java stuff will return a hashCode or something.  Basic types Int, Boolean 

```
content = HtmlContent(messages(s"$key.hint", dynamicYear))
```

messages is 
```scala
def apply(key: String, args: Any*): String
```

So it would take the Int and call toString

Which is the same as 

```
val a = 2004
"$a"

```